hypothesis-python/RELEASE.rst

```diff
@@ -0,0 +1,5 @@
+RELEASE_TYPE: minor
+
+This release adds the ability to pass any object to :func:`~hypothesis.note`, instead of just strings. The pretty-printed representation of the object will be used.
+
+See also :issue:`3843`.
```

Comment:
debatable whether this should be minor or patch. Feel free to change either way.

Reply:
I think `minor` is correct on the principle that this is a feature which (slightly) changes our API 👍